src/ExportPro.Common/ExportPro.Common.Shared/Behaviors/LoggingBehavior.cs

```diff
@@ -0,0 +1,42 @@
+﻿
+
+using ExportPro.Common.Shared.Library;
+using MediatR;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+
+namespace ExportPro.Common.Shared.Behaviors;
+
+public class LoggingBehavior<TRequest, TResponse> 
+    : IPipelineBehavior<TRequest, TResponse>
+    where TRequest: IRequest<TResponse>
+    where TResponse: BaseResponse
+{
+
+    private readonly ILogger<LoggingBehavior<TRequest, TResponse>> _logger;
+    //TODO gonna need this when we have built auth
+    //private readonly IHttpContextAccessor _contextAccessor;
+
+    public LoggingBehavior(ILogger<LoggingBehavior<TRequest, TResponse>> logger)
```

Comment:
please use primary constructor

Reply:
do it eveywhere we can do it, also use namespces without braces, where we use it